appveyor.yml

```diff
@@ -71,7 +71,6 @@ build: false
 test_script:
   - cargo build --release --target %TARGET% --locked
   - cargo test --release --target %TARGET%
-  - cargo fmt --all -- --check
```

Comment:
`cargo fmt` can be stymied by conditional compilation and since we have some conditional compilation into Windows, we should retain this just in case.